components/tikv_util/src/config.rs

```diff
@@ -689,6 +689,88 @@ pub fn check_max_open_fds(expect: u64) -> Result<(), ConfigError> {
     }
 }
 
+#[macro_export]
```

Comment:
Right now this macro is standing between `#[cfg(unix)] pub fn check_max_open_fds` and `#[cfg(not(unix))] pub fn check_max_open_fds`. You should move it to the end.